src/views/collection-view/helpers.ts

```diff
@@ -3,18 +3,22 @@ import { splitProductFromFilename } from 'views/tutorial-view/utils'
 /**
  * takes db slug format --> waypoint/intro
  * and turns it to --> waypoint/tutorials/get-started-docker/intro
+ *
+ * We want to make sure to use the collection product in the path as
+ * that sets the proper product context. The tutorial db slug may
+ * reference a different product context
  */
 
 export function getTutorialSlug(
-  dbslug: string,
-  collectionSlug: string
+  tutorialDbSlug: string,
+  collectionDbSlug: string
 ): string {
-  const [product, tutorialFilename] = dbslug.split('/')
-  const collectionFilename = splitProductFromFilename(collectionSlug)
+  const [product, collectionFilename] = collectionDbSlug.split('/')
```

Comment:
This was actually a bug, we always want the product reference to come from the collection slug, since a tutorial slug may live within a different product directory. 

For example, take the tutorial slug `consul/get-started`. This could be included in a vault collection: `vault/integrations`. So we need the collection product to write the dev portal path properly.